Dockerfile

```diff
@@ -97,36 +77,36 @@ RUN ln -s "/usr/bin/g++" "/usr/bin/musl-g++"
 # necessarily the right ones) in an effort to compile OpenSSL 1.1's "engine"
 # component. It's possible that this will cause bizarre and terrible things to
 # happen. There may be "sanitized" header
-RUN echo "Building OpenSSL" && \
+RUN echo "Building OpenSSL ${OPENSSL_VERSION}" && \
```

Comment:
I think it's a great idea to add the version numbers!

As for variable syntax, according to the spec:

> Environment variables are notated in the Dockerfile either with $variable_name or ${variable_name}. They are treated equivalently and the brace syntax is typically used to address issues with variable names with no whitespace, like ${foo}_bar.

This project prefers `$variable_name` when unambiguous, and only uses `${variable_name}` when the next character would otherwise be interpreted as part of the name. Let's leave this as is.